examples/vortexrings/vortexrings_functions.jl

```diff
@@ -556,7 +555,7 @@ function calc_vorticity!(pfield, ws, Xs, xoRs, nrings, Z, R, probedir;
             ws[1, pi, ri] = vpm.get_W1(P)
             ws[2, pi, ri] = vpm.get_W2(P)
             ws[3, pi, ri] = vpm.get_W3(P)
-            Xs[:, pi, ri] .= P.X
+            Xs[:, pi, ri] .= P[1:3]
```

Comment:
should use a view instead of allocating [1:3]

Reply:
Resolved in [2a3435a](https://github.com/byuflowlab/FLOWVPM.jl/pull/13/commits/2a3435ae13ee51e563ecc985dd8daa266c1e95cd)